entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md

```diff
@@ -19,6 +19,414 @@ EF7 is also available as [daily builds](https://github.com/dotnet/efcore/blob/ma
+
+This query generates the following SQL:
+
+```sql
+SELECT CAST(JSON_VALUE([a].[Contact],'$.Address.Postcode') AS nvarchar(max))
+FROM [Authors] AS [a]
+WHERE CAST(JSON_VALUE([a].[Contact],'$.Address.City') AS nvarchar(max)) = N'Chigley'
+```
+
+Here's an example that does a bit more in the filter and projection, and also orders by the phone number in the JSON document:
+
+<!--
+        var orderedAddresses = await context.Authors
+            .Where(
+                author => (author.Contact.Address.City == "Chigley"
+                           && author.Contact.Phone != null)
+                          || author.Name.StartsWith("D"))
+            .OrderBy(author => author.Contact.Phone)
+            .Select(
+                author => author.Name + " (" + author.Contact.Address.Street
+                          + ", " + author.Contact.Address.City
+                          + " " + author.Contact.Address.Postcode + ")")
+            .ToListAsync();
+-->
+[!code-csharp[OrderedAddresses](../../../../samples/core/Miscellaneous/NewInEFCore7/JsonColumnsSample.cs?name=OrderedAddresses)]
+
+This query generates the following SQL:
+
+```sql
+SELECT (((((([a].[Name] + N' (') + CAST(JSON_VALUE([a].[Contact],'$.Address.Street') AS nvarchar(max))) + N', ') + CAST(JSON_VALUE([a].[Contact],'$.Address.City') AS nvarchar(max))) + N' ') + CAST(JSON_VALUE([a].[Contact],'$.Address.Postcode') AS nvarchar(max))) + N')'
+FROM [Authors] AS [a]
+WHERE (CAST(JSON_VALUE([a].[Contact],'$.Address.City') AS nvarchar(max)) = N'Chigley' AND CAST(JSON_VALUE([a].[Contact],'$.Phone') AS nvarchar(max)) IS NOT NULL) OR ([a].[Name] LIKE N'D%')
+ORDER BY CAST(JSON_VALUE([a].[Contact],'$.Phone') AS nvarchar(max))
+```
+
+And when the JSON document contains collections, then these can be projected out in the results:
+
+<!--
+        var postsWithViews = await context.Posts.Where(post => post.Metadata!.Views > 3000)
+            .AsNoTracking()
+            .Select(
+                post => new
+                {
+                    post.Author!.Name,
+                    post.Metadata!.Views,
+                    Searches = post.Metadata.TopSearches,
+                    Commits = post.Metadata.Updates
+                })
+            .ToListAsync();
+-->
+[!code-csharp[PostsWithViews](../../../../samples/core/Miscellaneous/NewInEFCore7/JsonColumnsSample.cs?name=PostsWithViews)]
+
+This query generates the following SQL:
+
+```sql
+SELECT [a].[Name], CAST(JSON_VALUE([p].[Metadata],'$.Views') AS int), JSON_QUERY([p].[Metadata],'$.TopSearches'), [p].[Id], JSON_QUERY([p].[Metadata],'$.Updates')
+FROM [Posts] AS [p]
+LEFT JOIN [Authors] AS [a] ON [p].[AuthorId] = [a].[Id]
+WHERE CAST(JSON_VALUE([p].[Metadata],'$.Views') AS int) > 3000
+```
+
+> [!NOTE]
+> More complex queries involving JSON collections require `jsonpath` support. Vote for [Support jsonpath querying](https://github.com/dotnet/efcore/issues/28616) if this is something you are interested in.
```

Comment:
Maybe add a note to the [SQL Server JSON index docs](https://docs.microsoft.com/en-us/sql/relational-databases/json/index-json-data?view=sql-server-ver16)?